docs/rules.md

```diff
@@ -49,6 +49,12 @@ hide:
     11. Do not write or install software on computers belonging to other players or servers without their informed consent.
         1.  When advertising malicious software, it should be made very clear that it is malicious during every interaction.
     12. Do not use No Chat Reports' chat encryption feature in public chat. Using it in private messages is allowed however, as long as the receiving party is able to decode it.
+    13. Any form of in game gambling must be fair to the players.
+        1. Gambling must *never* be done with (currency bought with) money that has value in the real world (FIAT, Crypto, etc.)
+        2. Any odds must be clearly displayed on an ingame monitor, poster or sign and immediately updated when changed, and in the user interface where you gamble.
```

Comment:
Lets lax this and say "updated before the next player comes in".

We can also make an exception for the user interface: in case it is too busy, it doesn't need to contain all the rules/odds, it can instead point to a nearby ingame place where they are.

Reply:
Thats an avenue for abuse, because if you have a couple open machines or the stats in a general area, it still could update without players knowing.

Reply:
Not quite, as long as the player is aware of the odds before they start gambling their savings away, there's no issue.